tests/netflix/test_cassandra_uncertain.py

```diff
@@ -108,7 +108,7 @@ def test_increasing_qps_simple():
         )
 
     # We should generally want cheap CPUs
-    assert all(r[0][0] in ("r", "m", "i") for r in result)
+    assert all(r[0][0] in ("r", "m", "i", "c") for r in result)
```

Comment:
I don't think we need to do it in this PR, but I think this kind of refactor exposes the fragility of direct family comparisons, it seems that we were missing a lot of the `c` shapes which were actually fine most of the time. @abersnaze let's think how we can refactor some of the "shape" check tests to be more robust to evolving hardware - I started down the path of cost / cpu_power / "at least as good as X cluster" / "just check resources" but clearly we have a lot of work to do

Reply:
Maybe the script that generates family shapes can pre-compute the likely lowest-cost prefix that the tests should expect to see.